src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -36,7 +35,9 @@ def create_configstore(client,
                        tags=None,
                        assign_identity=None,
                        enable_public_network=None,
-                       disable_local_auth=None):
+                       disable_local_auth=None,
+                       retention_days=7,
+                       enable_purge_protection=None):
```

Comment:
>enable_purge_protection=None [](http://example.com/codeflow?start=23&length=28)

Should the default be False?

Reply:
I think it `False` is more robust. Updated.